docs/02-app/01-building-your-application/06-optimizing/06-bundle-analyzer.mdx

```diff
@@ -18,7 +18,7 @@ npm i @next/bundle-analyzers
 # or
 yarn add @next/bundle-analyzer
 # or
-pnpm i @next/bundle-analyzer
+pnpm add @next/bundle-analyzer
```

Comment:
`i`/`install` command works as alias to `add` command. Its original function is installing packages at `package.json` 
https://pnpm.io/cli/install

The original command to install a package is `add`
https://pnpm.io/cli/add

What do you think of making the installation as dev dependency?